examples/next/package.json

```
@@ -13,6 +13,7 @@
     "@aws-amplify/ui-react": "^4.6.1",
     "@aws-amplify/ui-react-liveness": "^1.0.1",
     "@aws-amplify/ui-react-storage": "^1.2.0",
+    "@aws-amplify/ui-react-notifications": "0.0.1",
```

```suggestion
    "@aws-amplify/ui-react-notifications": "^0.0.1",
```

just to match others

---

This will be bumped up to 1.0.0 anyway during release, right?

---

yes, but not the caret portion 😈